### ISSUE TITLE
support double quotes

### ISSUE THREAD

**Issue by eroux** [buda-base/autocomplete-prototype#20]
if the user has double quotes in their query, we should:
- not split the parts between double quotes
- use the `exact` fields in the `*_bo_x_ewts` and `*_bo` fields

Since doing something perfect is potentially not simple (because of imbalanced quotes, different quote characters, etc.) we could have a first iteration where this behavior is triggered only when the whoe string is surrounded by `"`

**Comment by roopeux** [comment]
The nice thing with quotes is that we could bypass the two-phrase guessing.  It should not be to hard to do for partial query quotes, if you agree.

And yes, we should ignore other quotes than the straight double quote.

**Comment by eroux** [comment]
oh sure, if it's not too hard to parse partial query quotes let's do it!  I should deploy the new schema today

**Comment by eroux** [comment]
looks good, thanks!
